components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/impl/AlertSubscriptionsApiServiceImpl.java

```diff
@@ -138,6 +163,78 @@ public Response getSubscribedAlertTypes(MessageContext messageContext) {
         return Response.status(Response.Status.OK).build();
     }
 
+    /**
+     * Retrieve a list of bot detection alert subscriptions
+     *
+     * @param messageContext
+     * @return list of bot detection alert subscriptions
+     * @throws APIManagementException if an error occurs when retrieving bot detection alert subscriptions
+     */
+    @Override
+    public Response getBotDetectionAlertSubscriptions(MessageContext messageContext) throws APIManagementException {
+
+        APIAdmin apiAdmin = new APIAdminImpl();
+        List<BotDetectionData> botDetectionDataList = apiAdmin.getBotDetectionAlertSubscriptions();
+        BotDetectionAlertSubscriptionListDTO listDTO =
+                BotDetectionMappingUtil.fromAlertSubscriptionListToListDTO(botDetectionDataList);
+        return Response.ok().entity(listDTO).build();
+    }
+
+    /**
+     * Subscribe for bot detection alerts
+     *
+     * @param body           email to be registered for the subscription
+     * @param messageContext
```

Comment:
Add a description or remove the line

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/9702/commits/f39ed1e5333a9182f3bae96a5b30ec0eee6bcbc9